gradle/libs.versions.toml

```diff
@@ -0,0 +1,153 @@
+# Also see settings.gradle file for dynamic versionCatalogs dependencies
+
+[versions]
+antlr = "4.9.3"
+deltaStandalone = "0.6.0"
+deltaCore = "2.2.0"
+flink115 = { strictly = "[1.15, 1.16[", prefer = "1.15.0"}
+flink116 = { strictly = "[1.16, 1.17[", prefer = "1.16.2"}
+flink117 = { strictly = "[1.17, 1.18[", prefer = "1.17.1"}
+guava = "32.1.1-jre"
+hadoop2 = "2.7.3"
+hive2 = { strictly = "[2, 3[", prefer = "2.3.9"}
+hive3 = "3.1.3"
+jackson211 = "2.11.4"
+jackson212 = "2.12.3"
+jackson213 = "2.13.4"
+jetty = "9.4.43.v20210629"
+junitJupiter = "5.9.2"
+mockito = "4.11.0"
+nessie = "0.63.0"
+orc = "1.9.0"
+parquet = "1.13.1"
+scalaCollectionCompat = "2.6.0"
+sparkHive31 = "3.1.3"
+sparkHive32 = "3.2.2"
+sparkHive33 = "3.3.2"
+sparkHive34 = "3.4.1"
+springBoot = "2.5.4"
+tez010 = "0.10.2"
+tez08 = { strictly = "[0.8, 0.9[", prefer = "0.8.4"}
+
+[libraries]
+activation = "javax.activation:activation:1.1.1"
+aircompressor = "io.airlift:aircompressor:0.24"
+aliyun-sdk-oss = "com.aliyun.oss:aliyun-sdk-oss:3.10.2"
+antlr-antlr4 = { module = "org.antlr:antlr4", version.ref = "antlr" }
+antlr-runtime = { module = "org.antlr:antlr4-runtime", version.ref = "antlr" }
+arrow-memory-netty = "org.apache.arrow:arrow-memory-netty:11.0.0"
+arrow-vector = "org.apache.arrow:arrow-vector:11.0.0"
+avro-avro = "org.apache.avro:avro:1.11.1"
+awssdk-bom = { module = "software.amazon.awssdk:bom", version = "2.20.18" }
+caffeine = "com.github.ben-manes.caffeine:caffeine:2.9.3"
+calcite-core = "org.apache.calcite:calcite-core:1.10.0"
+calcite-druid = "org.apache.calcite:calcite-druid:1.10.0"
```

Comment:
wouldn't it make sense to extract this into a version and use it for both libs?

Reply:
done